framework/doc/content/source/mfem/bcs/MFEMBoundaryIntegratedBC.md

```diff
@@ -1,10 +1,10 @@
-# MFEMScalarBoundaryIntegratedBC
+# MFEMBoundaryIntegratedBC
```

Comment:
I should have commented on this before. Why have `Boundary` in the name with the `BC` suffix?